src/SFA.DAS.AssessorService.ApplyTypes/Roatp/GatewayReviewStatus.cs

```diff
@@ -5,8 +5,8 @@ public static class GatewayReviewStatus
         public const string Draft = "Draft";
         public const string New = "New";
         public const string InProgress = "In Progress";
-        public const string Approved = "Approved";
-        public const string Declined = "Declined";
-        public const string Clarification = "Clarification";
+        public const string Pass = "Pass";
+        public const string Decline = "Decline";
+        public const string AskForClarification = "Ask for clarification";
```

Comment:
As mentioned in the Apply PR, this string is too long for the field in the Apply table

Reply:
I have increased it to 50 characters (look in src/SFA.DAS.ApplyService.Database/Tables/Apply.sql file), so it will not get truncated. But it is also possible "Ask for clarification" to be changed to "Clarification sent" after our Statuses meeting.